driver.cpp

```diff
@@ -0,0 +1,100 @@
+#include "split_service.h"
+
+int main()
+{
+   SplitService *ser = new SplitService(4);
+
+   User *U1 = new User(0, "u1", "Susmita", "sm@com", "11890");
+   ser->set_user_table("u1", U1);
+
+   User *U2 = new User(1, "u2", "Ayan", "ay@com", "11890");
+   ser->set_user_table("u2", U2);
+
+   User *U3 = new User(2, "u3", "Moumita", "mm@com", "11890");
+   ser->set_user_table("u3", U3);
+
+   User *U4 = new User(3, "u4", "Subhayan", "sb@com", "11890");
+   ser->set_user_table("u4", U4);
+
+   int cont = 0;
+   do{
+        cout << "Enter operation " << endl;
+        string op;
+        cin >> op;
+        if (op == "SHOW")
+        {
+           cout << "for a specific user?" << endl;
+           string ans;
+           cin >> ans;
+           if (ans == "YES" || ans == "Yes" || ans == "yes")
```

Comment:
Should follow the same format as mentioned in the problem statement at least during the interview.